frontend/pages/[resourceType].tsx

```diff
@@ -104,8 +102,24 @@ export const getServerSideProps: GetServerSideProps<{
     // Measure Repository should not provide a bundle without an entry
     throw new Error('Measure Repository bundle has no entry.');
   }
-  const ids = bundle.entry.map(entry => entry.resource?.id);
+  const resources = bundle.entry as fhir4.BundleEntry<FhirArtifact>[];
```

Comment:
Not related to your code but there's a double equal string comparison above. Can you change that to `!==`?